examples/quota-increase/README.md

```diff
@@ -0,0 +1,11 @@
+# Request AWS Quota Increase
```

Comment:
Please add a test! At the very least, it should run `plan`. That said, if it's OK to request the same quota multiple times concurrently, then the test could run `apply` and `destroy`.

Reply:
I had no idea how to test this.
> if it's OK to request the same quota multiple times concurrently, then the test could run apply and destroy.

Just realized it is! This way I can check if the `quota_name` and `service_name` are correct!

Reply:
Changes made at af98666